kadi/commands/tests/test_commands.py

```diff
@@ -314,6 +316,32 @@ def test_commands_create_archive_regress(tmpdir, version_env):
                 # Path("cmds_local.txt").write_text(out)
                 assert len(cmds_flight) == len(cmds_local)
 
+            # Roughly validate NSM quaternions and remove from later comparison
+            nsm_idxs = np.flatnonzero(cmds_flight['npnt_enab'] == False) # noqa
+            for idx in nsm_idxs:
+                flight_att = cmds_flight[idx]['params']['targ_att']
+                flight_q = Quat(q=flight_att)
+                local_q = Quat(q=cmds_local[idx]['params']['targ_att'])
+                delta_point = sphere_dist(flight_q.ra, flight_q.dec,
+                                          local_q.ra, local_q.dec)
+
+                assert delta_point < 0.1
+                assert np.isclose(flight_q.roll, local_q.roll, atol=0.1)
+
+                for cmd in (cmds_local[idx], cmds_flight[idx]):
+                    del cmd['params']['targ_att']
+
+                # Not sure if it would be OK to just remove all 'prev_att'
```

Comment:
Why not do the same `dq` comparison for `prev_att` as for `tarq_att` in a loop?

Reply:
I'm not entirely sure what you mean.  The 'prev_atts' should be the same as the 'targ_att', just later, so I didn't bother to do the same check, whatever it is, on the numbers.  As to why I didn't loop through and just check them all, we can do that too. 

The differences only came up on the npnt_enab == False cmds for 'targ_att' and I couldn't pull out the next commands that referenced the same off attitudes easily.

Reply:
> The 'prev_atts' should be the same as the 'targ_att'

That is the hope but it needs to be verified. Basically for a single `flight` command you can think of `prev_att` and `targ_att` as two unrelated quaternions that needs to be checked against the corresponding `local` versions. So it's just something like:
```
            nsm_idxs = np.flatnonzero(cmds_flight['npnt_enab'] == False) # noqa
            for idx in nsm_idxs:
                for att_name in ('prev_att', 'targ_att'):
                    flight_att = cmds_flight[idx]['params'][att_name]
                    flight_q = Quat(q=flight_att)
                    local_q = Quat(q=cmds_local[idx]['params'][att_name])
                    ...
```
    .

Reply:
The issue really though is that the prev_att that is the known problem does not correspond to one that matches npnt_enab == False.  So it becomes a question about if we want to use this complicated logic to isolate to just the targ_att when npn_enab == False and the prev_att when it matches a problematic targ_att, or if we want to just open this up to do a better numeric comparison instead of == for all quaternions in the rows.